src/examples/SimpleWizard/SimpleWizard.code.js

```diff
@@ -0,0 +1,25 @@
+import { indent } from "../../utils";
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { SimpleWizard } from "!!./SimpleWizard";
+
+export const simpleWizardCode = `
+<head>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio-script.prod.min.js"></script>
+</head>
+<body>
+  <script>
+${indent(
+  SimpleWizard.toString().replaceAll(
+    "_helpers_FormioLoader__WEBPACK_IMPORTED_MODULE_0__",
```

Comment:
why do you need to do this?

Reply:
this code will show in the code snippet of the story, provides a functional example if the user copy and paste the code to a html page

Reply:
the function has alredy transpiled by webpack, and webpack has replaced `FormioLoader` with `_helpers_FormioLoader__WEBPACK_IMPORTED_MODULE_0__`, so i have to convert it back...